examples/Console/TestConsoleExporter.cs

```diff
@@ -102,18 +101,6 @@ internal static object Run(ConsoleOptions options)
 
         internal class MyProcessor : ActivityProcessor
         {
```

Comment:
Writing own Processor is also greatly simplified now. No need of worrying about "next".